srv/salt/ceph/rbd/benchmarks/cleanup_master.sls

```diff
@@ -1,6 +1,12 @@
-delete benchmark image:
+delete benchmark pool:
   cmd.run:
-    - name: rbd rm deepsea_benchmark
+    - name: ceph osd pool rm {{ salt['pillar.get']('rbd_benchmark_pool', 'deepsea_rbd_benchmark') }} {{ salt['pillar.get']('rbd_benchmark_pool', 'ds_rbd_bench_pool') }} --yes-i-really-really-mean-it
```

Comment:
This depends on having the mon setting that allows pools to be deleted

Reply:
We could inject this flag, remove and reinstate the default? Though this would get more complex if the setting already exists.